.devcontainer/make_devcontainers.sh

```diff
@@ -82,7 +82,7 @@ if [ "$VERBOSE" = true ]; then
 fi
 
 # Read matrix.yaml and convert it to json
-matrix_json=$(yq -o json ${MATRIX_FILE})
+matrix_json=$(yq . ${MATRIX_FILE})
```

Comment:
I made this change because of an error I got while running this script. It seems the `-o json` option doesn't exist.

```
jq: Unknown option -o
Use jq --help for help with command-line options,
or see the jq manpage, or online docs  at https://jqlang.github.io/jq
yq: Error running jq: BrokenPipeError: [Errno 32] Broken pipe.
```

Reply:
Hm, this must be a yq version difference thing or something.

Reply:
Yeah. I tried to read the changelogs and find the difference but I wasn't successful.

Reply:
Looks like CI needs the previous code so I'll revert this. Maybe my local `yq` is out of date. It's `3.2.3` from conda-forge. Normally conda-forge has newer versions of everything...

Reply:
```suggestion
matrix_json=$(yq -o json ${MATRIX_FILE})
```

Reply:
Aw man! It's even worse.

The conda-forge package `yq` is https://github.com/kislyuk/yq, but the tool we want is https://github.com/mikefarah/yq. These are not the same tool! :(

Reply:
For my future self: I would need to install the conda package `go-yq` to get the tool I want.